client_plugin/vmdk_plugin/main_windows.go

```diff
@@ -20,36 +20,37 @@ import (
     log "github.com/Sirupsen/logrus"
 )
 
-var logger service.Logger
+// pluginServer is a wrapper for vDVS plugin so that the plugin can be run
+// as a service on Windows. Windows controls services by setting up callbacks
+// that is non-trivial. This wrapper utilizes a vendor library to handle this.
+type pluginServer struct{}
```

Comment:
I will change it to PluginService then. Let me know if you have better suggestions.